src/test/java/org/opentripplanner/transit/raptor/moduletests/B04_AccessEgressNoWaitTest.java

```diff
@@ -0,0 +1,140 @@
+package org.opentripplanner.transit.raptor.moduletests;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.opentripplanner.transit.raptor._data.transit.TestRoute.route;
+import static org.opentripplanner.transit.raptor._data.transit.TestTransfer.walk;
+import static org.opentripplanner.transit.raptor._data.transit.TestTripSchedule.schedule;
+import static org.opentripplanner.transit.raptor.api.request.RaptorProfile.MULTI_CRITERIA;
+import static org.opentripplanner.transit.raptor.api.request.RaptorProfile.NO_WAIT_STD;
+import static org.opentripplanner.transit.raptor.api.request.RaptorProfile.STANDARD;
+import static org.opentripplanner.transit.raptor.api.request.SearchDirection.REVERSE;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.transit.raptor.RaptorService;
+import org.opentripplanner.transit.raptor._data.RaptorTestConstants;
+import org.opentripplanner.transit.raptor._data.api.PathUtils;
+import org.opentripplanner.transit.raptor._data.transit.TestTransitData;
+import org.opentripplanner.transit.raptor._data.transit.TestTripSchedule;
+import org.opentripplanner.transit.raptor.api.request.RaptorRequestBuilder;
+import org.opentripplanner.transit.raptor.rangeraptor.configure.RaptorConfig;
+
+/**
+ * FEATURE UNDER TEST
+ *
+ * This test focus on testing access and egress with basic Raptor - NOT Range Raptor. All tests are
+ * run with one iteration only. The Range Raptor make sure the best access/egress is used, but
+ * without it, we should board at the first possible stop for the standard raptor, but not for the
+ * no-wait-std. The reason for this is that the standard optimize on best arrival time, while the
+ * no-wait-std optimize on finding the shortest travel time.
```

Comment:
Same clarity problems as in the other test file.

Reply:
This is fixed in 51aa580 together with the cleanup of he routing strategy naming.